tests/steps/steps.py

```diff
@@ -404,19 +421,21 @@ def send_msig_txn(context):
 
 @then("the transaction should go through")
 def check_txn(context):
-    last_round = context.acl.status()["lastRound"]
+    send_zero_transactions(context)
+    # last_round = context.acl.status()["lastRound"]
```

Comment:
recommend removal of commented out code in this function

Reply:
Good point - I'm in favor or removing these lines because there are other steps that do the same thing in this SDK, e.g. 
`I can get the transaction by ID`